libs/platform/src/lib/components/list/list.component.ts

```diff
@@ -0,0 +1,631 @@
+        } else {
+            this.items = this.dsItems;
+        }
+        this.id = `fdp-list-${nextListId++}`;
+        // using selection Model for multiselect
+        if (this.selectionMode === 'multi') {
+            this.multiSelect = true;
+            this.ariaMultiselectable = true;
+        } else { this.multiSelect = false; }
+        this.selectionModel = new SelectionModel<BaseListItem>(
+            this.multiSelect,
+            this.selectedItems
+        );
+
+        this.selectionModel.changed.subscribe(e => {
+            this.selectedItems = this.selectionModel.selected;
+            const event = new SelectionChangeEvent();
+            event.selectedItems = this.selectedItems;
+            this.selectedItemChange.emit(event);
+        });
+
+
+    }
+
+    /** @hidden */
+    /**Keyboard manager on list items, set values when passed via array */
+    ngAfterViewInit(): void {
+
+        this.keyManager = new FocusKeyManager<BaseListItem>(this.ListItems).withWrap();
+        this.ListItems.forEach((item) => {
+            item.navigated = this._navigated;
+            item.navigationIndicator = this._navigationIndicator;
+            item.contentDensity = this.contentDensity;
+            item.selectionMode = this.selectionMode;
+            item.listType = this.listType;
+            item.hasByLine = this._hasByLine;
+            item.noSeperator = this.noSeperator;
+        });
+    }
+
+
+    /** @hidden */
+    /**Setting values from list to list items
+     * example:
+     * Does list item has navigation,
+     * should show arrows,
+     * will it be compact mode,
+     * should be in which selection mode
+     * set values when passed via datasource
+     */
+    ngAfterContentInit(): void {
+        this._itemsSubscription = this.ListItems.changes.subscribe((items) => {
+            items.forEach((item) => {
+                item.navigated = this._navigated;
+                item.navigationIndicator = this._navigationIndicator;
+                item.contentDensity = this.contentDensity;
+                item.selectionMode = this.selectionMode;
+                item.listType = this.listType;
+                item.hasByLine = this._hasByLine;
+                item.noSeperator = this.noSeperator;
+            });
+        });
+    }
+
+    ngOnDestroy(): void {
+        if (isDataSource(this.dataSource)) {
+            this.dataSource.close();
+        }
+        if (this._dsSubscription) {
+            this._dsSubscription.unsubscribe();
+        }
+        if (this.selectionModel) {
+            this.selectionModel.changed.unsubscribe();
+        }
+        if (this._itemsSubscription) {
+            this._itemsSubscription.unsubscribe();
+        }
+    }
+
+    get value(): any {
+        return this.selectedItems;
+    }
+    set value(value: any) {
+        this.selectedItems = value;
+        this._changeDetectorRef.markForCheck();
```

Comment:
please see set value from parent

Reply:
you need to inform parent.

Reply:
updated here and in all applicable places for list.componen.ts